modules/tensor_mechanics/src/materials/abaqus/AbaqusUMATStress.C

```diff
@@ -113,14 +116,22 @@ AbaqusUMATStress::AbaqusUMATStress(const InputParameters & parameters)
   _aqDPRED.resize(_number_external_fields + _number_external_properties);
 }
 
+void
+AbaqusUMATStress::initialSetup()
+{
+  // if any of these optional properties are not available, we need to check the
```

Comment:
"if any of these optional properties are not available", i.e. if this is not an incremental finite strain simulation.

Reply:
Actually you could spell out the reason why this is done in the comment if you push again.